jupyter_server/services/events/handlers.py

```diff
@@ -75,6 +79,45 @@ def on_close(self):
         self.event_bus.handlers.remove(self.logging_handler)
 
 
+class EventHandler(APIHandler):
+    """REST api handler for events"""
+
+    auth_resource = AUTH_RESOURCE
+
+    @property
+    def event_bus(self):
+        """Jupyter Server's event bus that emits structured event data."""
+        return self.settings["event_bus"]
```

Comment:
This block shouldn't be needed since it's already in one of the base classes:
https://github.com/jupyter-server/jupyter_server/blob/f25fd33ae068e55039905013f1552ff5ea051e37/jupyter_server/base/handlers.py#L343-L345